### PR TITLE
Adds a JSCS config and testing.

### PR DESCRIPTION
**Pull request by qubyte**

### CODE REVIEW

**Comment by allouis** [comment]
:+1:

File: test.js [hunk 1]
```diff
@@ -60,7 +59,7 @@ define(['hermes', 'chai'], function (hermes, chai) {
             });
 
             describe('a notification', function () {
-                it('should create a list item', function(){
+                it('should create a list item', function () {
```
Comment (lcapel) [inline test.js]: :expressionless:

**Comment by lcapel** [comment]
:+1:

File: test.js (hunk 1 above)
Comment (qubyte) [inline test.js]: This was prior to our newer practice of not using "should" in tests.

Or is it whitespace? My repo, my rules. ;)

**Comment by lcapel** [comment]
Don't have write access so cannot merge this :confused: Oh well...

**Comment by qubyte** [comment]
Done. Thanks for the reviews! :D

**Pull request merged**